### PR TITLE
Ensure minimum 2 replicas of console

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Fixes issue #83 

@jwforres @spadgett

### CODE REVIEW

File: pkg/console/operator/operator.go [hunk 1]
```diff
@@ -274,8 +274,8 @@ func (c *ConsoleOperator) defaultConsole() *consolev1alpha1.Console {
 				// if Verison is not 4.0.0 our reconcile loop will not pick it up
 				Version: "4.0.0",
 			},
-			// one replica is created
-			Count: 1,
+			// minimum of 2 replicas
+			Count: 2,
```
Comment (enj) [inline pkg/console/operator/operator.go]: Uh why is this being changed in two places?  And why is `2` instead of `3`?

File: examples/cr.yaml [hunk 2]
```diff
@@ -15,12 +15,9 @@ spec:
   # - Unmanaged indicates that the operator should idle and ignore the console and its sub-resources
   # - Removed indicates that the operator should delete all of the sub-resources of the console
   managementState: Managed
-  # there should be only one console at this time
-  # this could be used to support leader election if desired
-  count: 1
+  # there should be the same number of console replicas as the number of masters, minimum 2
```
Comment (wking) [inline examples/cr.yaml]: Looks like this (and the PR title and commit message) missed [the bump from 2 to 3][1].

[1]: https://github.com/openshift/console-operator/pull/86#discussion_r239562790

Comment (benjaminapetersen) [reply examples/cr.yaml]: Thanks @wking!  Sorry about that, rushed it in for demo sake, should have stopped & updated the extra bits. :/

I'll follow a PR to fix the example file at least.

Comment (benjaminapetersen) [reply examples/cr.yaml]: https://github.com/openshift/console-operator/pull/101

**Pull request merged**